requestmanager/asyncloader/asyncloader_test.go

```diff
@@ -391,13 +391,15 @@ func withLoader(st *store, exec func(ctx context.Context, asyncLoader *AsyncLoad
 }
 
 func assertSuccessResponse(ctx context.Context, t *testing.T, resultChan <-chan types.AsyncLoadResult) {
+	t.Helper()
```

Comment:
I am curious would this still print out the line at which an assertion fails if the assertion is in a helper function?

Reply:
Right now, if the assertion on line 397, the error message will point to `asyncloader_test.go:397`, so it's not going to tell you what part of the original test failed.

Presumably you'll still have all the information you need about the failure via the context given by the assertion, like `"should send response"`, so the position within the helper function shouldn't be necessary.

Reply:
In case I wasn't clear: the file position it will give you now is where the assertSuccessResponse call happened, such as inside a `func TestFoo(t *testing.T)`.